packages/calypso-products/src/plans-list.tsx

```diff
@@ -3622,11 +3623,11 @@ if ( isEnabled( 'plans/migration-trial' ) ) {
 
 PLANS_LIST[ PLAN_HOSTING_TRIAL_MONTHLY ] = {
 	...getPlanBusinessDetails(),
-	type: TYPE_BUSINESS,
+	type: TYPE_HOSTING_TRIAL,
 	group: GROUP_WPCOM,
 	getProductId: () => 1058,
 	getPathSlug: () => PLAN_HOSTING_TRIAL_MONTHLY,
-	term: TERM_MONTHLY,
+	term: TERM_ANNUALLY,
```

Comment:
Was this change intentional, @p-jackson? There is no tenure associated with the FT plan, so does it matter?

Reply:
Oops, just saw that: "The trial plan had to be set to an annual term because the plans grid crashes when there's no annual term for a plan. I think there's a hard coded assumption that all plans must have an annual variant at least. But that's not true for us. We should see if we can fix this underlying assumption."

Please ignore!

Reply:
> The trial plan had to be set to an annual term because the plans grid crashes when there's no annual term for a plan. I think there's a hard coded assumption that all plans must have an annual variant at least. But that's not true for us. We should see if we can fix this underlying assumption.

In the database it's a monthly plan (even though we manually adjust the expiry date to make it a week), so I feel like it'd be more consistent to have the term be `TERM_MONTHLY` in Calypso too. So that database definition matches the Calypso definition.

But yeah it's a hack to get things going. Maybe you can find a way to fix the plans grid to support products with no annual term.